### PR TITLE
Beta explorer cll

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Public beta for Explorer's column-level lineage feature

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Needs review from product team

Adding pages:
- [x] Add/remove page in `website/sidebars.js`
- [x] Provide a unique filename for new pages

### CODE REVIEW

File: website/docs/docs/collaborate/column-level-lineage.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+---
+
+# Column-level lineage <Lifecycle status='beta' />
+
+dbt Explorer now offers column-level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a resource (model, source, or snapshot) in a dbt project, Explorer provides end-to-end lineage for the data in that column given how it's used.
+
+CLL is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the [Discovery API](/docs/dbt-cloud-apis/discovery-api).
+
+:::tip Check out our beta
+Explorer's CLL is currently available as a [public beta](/docs/dbt-versions/product-lifecycles#dbt-cloud) for Enterprise plans. Please check it out! We'd love to [know what you think](https://docs.google.com/forms/d/e/1FAIpQLSdpCbVkGY9QwfExFonpWE4DTOKi3fQxBGLD0wwKYpkMjgcE7g/viewform)!
```
Comment (cafzal) [inline website/docs/docs/collaborate/column-level-lineage.md]: @nghi-ly "Explorer's CLL is currently available as a [public beta](/docs/dbt-versions/product-lifecycles#dbt-cloud) for Enterprise plan accounts."

**Review by cafzal — CHANGES_REQUESTED** [review]
minor edit

File: website/docs/docs/collaborate/column-level-lineage.md (hunk 1 above)
Comment (nghi-ly) [inline website/docs/docs/collaborate/column-level-lineage.md]:
```suggestion
Explorer's CLL is currently available as a [public beta](/docs/dbt-versions/product-lifecycles#dbt-cloud) for Enterprise plan accounts. Please check it out! 
```

**Review by cafzal — APPROVED** [review]
thanks!

**Pull request merged**